Gruntfile.js

```diff
@@ -21,11 +21,8 @@ module.exports = function( grunt ) {
 			"src/intro.js",
 			"src/version.js",
 			"src/migrate.js",
-			"src/attributes.js",
 			"src/core.js",
 			"src/css.js",
-			"src/data.js",
```

Comment:
I may leave it empty if you wish but adding new modules isn't that hard anyway. :)